working-together/README.md

```diff
@@ -221,10 +221,18 @@ We've compiled some guidelines from various sources to help people feel included
   - Try to be aware of others' comfort level with eye contact.
   - Err on the side of less eye contact.
 
+## Inclusive language guidelines
 
-## Pronoun guidelines
+- **Acronyms**
+  - The use of acronyms makes it hard for someone who doesn't know their meaning to engage in conversations or have total clarity on some code.
+  - Avoid using acronyms in code, conversations, or documentation as much as possible.
+  - If the use of specific acronyms is inevitable for any reason...
+    - Make sure to document their meanings so that everyone can see them anytime.
+    - When mentioning them in conversations, check if people involved in the conversation know what they mean.
 
-At thoughtbot, we strive to make everyone feel included and able to bring their entire self to work. One of the ways we do this is by providing space for people to feel comfortable sharing their pronouns. Pronouns are often shared during introductions and can also be added to your Slack profile, Hub profile, Zoom display name, email signature, etc.
+### Pronoun guidelines
+
+At thoughtbot, we strive to make everyone feel included and able to show up as the most authentic self that they feel comfortable bringing to work. One of the ways we do this is by providing space for people to feel comfortable sharing their pronouns. Pronouns are often shared during introductions and can also be added to your Slack profile, Hub profile, Zoom display name, email signature, etc.
```

Comment:
Sorry! I realize I committed to the change rather than offering a suggestion. You can revert the change if you prefer that I add a suggestion.

The change that I made was updating the language from "to be able to bring their entire self to work" to "to be able to show up as the most authentic self that they feel comfortable bringing to work." 

Bringing your whole self to work is a phrase that's become popular across DEI as a way to be inclusive but there is nuance around this phrase. Some folks want to keep their personal and professional lives separate. And for marginalized folks, it's historically caused harm in different work contexts when trying to bring their authentic selves. Therefore, we want to phrase it more as a choice for folks and how comfortable they may feel bringing their full selves.